services/can_logger/can_logger.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+import socket
+import postgres
+import csv
+import os
+
+def write_to_csv(timestamp, can_id, can_data, host_interface):
+
+
+    with open('/data/log/' + host_interface + '.csv', mode = 'a') as log:
+
+        log = csv.writer(log, delimiter = ',', quotechar = '"',
```

Comment:
`csv_init()` might also want to make the `csv.writer()` object and pass that here instead. I am worried about calling `write_to_csv()` as is many times a second but I could be wrong

Reply:
Indeed. This could be written more efficiently.